www/src/slices/mapper/LeafletMap.tsx

```diff
@@ -84,6 +84,9 @@ export let LeafletMap = () => {
   // redraw boundary for change of `framework`
   useEffect(() => {
     dispatch(mapperActions.selectPolygon(undefined))
+    if (isCurrentZoomDefault(state.zoom)) {
+      map.setView(frameworks[state.query.framework].defaultQuery.center, state.zoom)  
```

Comment:
Also there is a flyTo method which looks a bit more swish but the small basemap means you can't centre properly so it kind of bounces you back which is a bit jarring. I might fiddle with this for fun at some point

Reply:
OK, so I take it we will commit this as-is for now